mlflow/models/evaluation/base.py

```diff
@@ -1278,7 +1291,20 @@ def evaluate(
           specified, unless the ``evaluator_config`` option **log_model_explainability** is
           explicitly set to ``True``.
 
-    :param model: A pyfunc model instance, or a URI referring to such a model.
+    :param model: One of the following:
+
+                  - A pyfunc model instance
+
+                  - A URI referring to a pyfunc model
+
+                  - A function. The function should have the same signature as the
+                    :py:func:`predict <mlflow.pyfunc.PyFuncModel.predict>` method, which has
+                    the following signature::
+
+                        predict(data: Union[pandas.(Series | DataFrame),
+                        numpy.ndarray, csc_matrix, csr_matrix, List[Any], Dict[str, Any], str],
+                        params: Optional[Dict[str, Any]] = None) →
+                        Union[pandas.(Series | DataFrame), numpy.ndarray, list, str]
```

Comment:
Can we include a code example instead of copying the exact signature? Otherwise, it becomes harder to keep this up to date